calico-enterprise/reference/installation/tigerastatus.mdx

```diff
@@ -0,0 +1,42 @@
+---
+description: Descriptions of Tigera Status fields
+---
+
+# Tigera status
+
+Installing {{prodname}} on your Kubernetes cluster is managed by the Tigera operator. The Tigera operator is deployed as a ReplicaSet in the `tigera-operator` namespace, and records status in a custom resource named, `tigerastatus`. The operator get its configuration from several Custom Resources (CRs); the central one is the Installation CR.
+
+To check tigerastatus, use the following command.
+
+`kubectl get tigerastatus`
+
+## Log storage controllers
+
+The following controllers comprise the log storage component. Note the following:
+
+- All controllers are required and are not configurable
```

Comment:
I don't think any of these bullet points are actually relevant to end-users, we can probably just omit them.

In fact, I don't think we should be referring to these as controllers at all really - a "controller" is an implementation detail. It just so happens that there are controllers backing these API resources.

Reply:
I added the text about being non-configurable because this was Pooriya's original question: are these resource configurable? But maybe with the new flow it will be more clear that they are not. Perhaps the addition of new status fields threw him off.

Reply:
Ah, I think this text through me off because it's talking about _controllers_ which are different than TigeraStatus resources.

I think it's fine to say that TigeraStatus is intended to be a read-only API that reports summarized product status.

Reply:
Fixed.